Yoast/javascript
# Create Reactified Toggle Component

## Summary
Creates the Toggle component based on the HTML and CSS provided by Team front-end.


This PR can be summarized in the following changelog entry:

* [yoast-components] Adds the Toggle component.

## Relevant technical choices:

* I have created a class component so that I can change the value of the input field. This value can be used to save it to the database / handled server-side.

## Test instructions

This PR can be tested by following these steps:
1. Start the example components app.
2. Make sure that the toggle works as expected.
3. Check that you can set the default selection used the `selected` prop.
4. Check that you get the selected value using the `onChange` prop.

## Quality assurance


Fixes #567
